core/templates/dev/head/pages/collection_player/CollectionPlayer.js

```diff
@@ -275,5 +280,16 @@ oppia.controller('CollectionPlayer', [
         $scope.generatePathParameters();
       }
     }, true);
+
+    $scope.scrollToLocation = function(id) {
+      $location.hash(id);
+      $anchorScroll();
+    };
+    // Touching anywhere outside the mobile preview should hide it.
```

Comment:
I agree that the function is reversed. I'm not sure what I must rename the variable to (now that it is just one var).